english/coffee/cookie/Walk/walk.md

```diff
@@ -0,0 +1 @@
+the walk opens up your mind and lets your creativity flow.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.